packages/wrangler/src/pages/download-config.ts

```diff
@@ -161,32 +183,105 @@ async function toEnvironment(
 	return configObj;
 }
 async function writeWranglerToml(toml: RawEnvironment) {
+	let tomlString = TOML.stringify(toml as TOML.JsonMap);
+
+	// Remove indentation from the start of lines, as this isn't common across TOML examples, and causes user confusion
+	tomlString = tomlString
+		.split("\n")
+		.map((line) => line.trimStart())
+		.join("\n");
+
 	// Pages does not support custom wrangler.toml locations, so always write to ./wrangler.toml
 	await writeFile(
 		"wrangler.toml",
-		[
-			`# Generated by Wrangler on ${new Date()}`,
-			TOML.stringify(toml as TOML.JsonMap),
-		].join("\n")
+		`# Generated by Wrangler on ${new Date()}\n${tomlString}`
 	);
 }
 
+function simplifyEnvironments(
+	preview: RawEnvironment,
+	production: RawEnvironment
+): {
+	topLevel: RawEnvironment;
+	preview?: RawEnvironment;
+	production?: RawEnvironment;
+} {
+	const topLevel = { ...preview };
+	// If the environments are equal, don't include a production override
```

Comment:
if we decide to not generate the config for both environments as per Pete's comment, I would argue that we should keep the `production` config and skip the `preview` override, since I would imagine that as a user the most important thing for me to know is what I am deploying to prod